Web_Development/FrontEnd/CSS_ANIMATION/README.md

```diff
@@ -0,0 +1,129 @@
+# CSS ANIMATION :--
+
+CSS(Cascading Style Sheets) is basically used to add designs and animations in the web pages without using JavaScript or Flash and make it more attractive to the users.
+
+<br/>
+
+*   CSS Animation Properties :--
+
+
+> 1. The first and the most important property of CSS Animation is **@keyframes** which is used to specify animation type. For example if the animation name is **First-Animation** then we need to write 
+
+
+```
+ @keyframes First Animation { css styles }
+
+```
+<br/>
+
+> 2. **Animation Delay** is another property which specify the starting timing of Animation.
+
+
+
+```
+div{
+    animation-delay: 5s;
+}
+
+```
+<br/>
+
+> 3. **Animation Duration** specify the timing for animation to complete once. It is also denoted in seconds.
+
+```
+div{
+    animation-duration: 10s;
+}
+
+```
+<br/>
+
+> 4. **Animation Direction** specify the direction of the animation such as forward, backward, alternate, reverse-alternate.
+
+```
+div{
+    animation-direction: normal/reverse/alternate/alternate-reverse;
+}
+
+```
+
+<br/>
+
+> 5. **Animation Iteration Count** specify the number of times the animation played.
+
+```
+div{
+    animation-iteration-count: 10;
+}
+
+```
+
+<br/>
+
+> 6. **Animation Timing Function** specify the speed type the animation played.
+
+```
+div{
+    animation-timing-function: linear/ ease/ ease-in/ ease-out/ ease-in-out;
+}
+
+```
+<br/>
+
```

Comment:
You need to add some more animation properties like Speed Curve 
You should try making this markdown more interactive, take a reference from the merged markdowns.

Reply:
Okk I'll do it